api/client/build.go

```diff
@@ -286,6 +291,13 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 	}
 	headers.Add("X-Registry-Config", base64.URLEncoding.EncodeToString(buf))
 
+	ulimitVar := flUlimits.GetList()
+	buf, err = json.Marshal(ulimitVar)
+	if err != nil {
+		return err
+	}
+	headers.Add("X-Docker-BuildUlimit", base64.URLEncoding.EncodeToString(buf))
```

Comment:
Can we use a get param ? I'm pretty sure we already have endpoint with the same get param key multiple time. other you could use the `docker/pkg/parsers/filters` pkg maybe ?